g_appcode.go

```diff
@@ -17,15 +17,14 @@ package main
 import (
 	"database/sql"
 	"fmt"
+	_ "github.com/go-sql-driver/mysql"
+	_ "github.com/lib/pq"
```

Comment:
Sometimes I use Emacs and it sorts the imports alphabetically automatically! You can move them down if you want.